examples/evaluation/evaluate_on_multiclass_classifier.py

```diff
@@ -13,8 +13,11 @@
 with mlflow.start_run() as run:
     model = LogisticRegression(solver="liblinear").fit(X_train, y_train)
     model_uri = mlflow.get_artifact_uri("model")
+    logged_model = mlflow.search_logged_models(
+        filter_string=f"source_run_id='{run.info.run_id}'", output_format="list"
+    )[-1]
```

Comment:
@dbczumar Can we add a function like `get_logged_model_by_name` to make it easier to fetch an autologged logged model?

Reply:
`mlflow/examples/mlflow-3/sklearn_autolog.py` uses the following code, but I think that's a bit complex and long:

https://github.com/mlflow/mlflow/blob/02af29b48d3611507c3e507270910f7a5e669f99/examples/mlflow-3/sklearn_autolog.py#L25-L30